.github/workflows/rust-wasmedge-macro-release.yml

```diff
@@ -5,62 +5,81 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  push:
-    branches:
-      - "master"
-      - "rust/release-wasmedge-macro"
-    paths:
-      - ".github/workflows/rust-wasmedge-macro-release.yml"
-      - "bindings/rust/wasmedge-macro/**"
-    tags:
-      - "rust-macro/[0-9]+.[0-9]+.[0-9]+*"
-  pull_request:
+  workflow_dispatch:
```

Comment:
Not sure why we give up the release trigger and change to manually trigger?

Reply:
The goal of the new design of the release workflows is to automate "everything", including adding tags, updating change log, bumping version, and etc.. As a result, triggering the release manually is the only thing we have to do when we wanna publish new crates and API docs. That would extremely reduce the original release job (git tag, git push, etc.) to a mouse click.

Reply:
Got it.